lib/pipefy_message/providers/aws_client/sqs_broker.rb

```diff
@@ -23,6 +23,11 @@ def initialize(opts = {})
           @is_staging = ENV["ASYNC_APP_ENV"] == "staging"
 
           queue_url = @sqs.get_queue_url({ queue_name: @config[:queue_name] }).queue_url
+
+          queue_url = queue_url.sub("http:", "https:")
+          # The string returned by the SQS client object is frozen,
+          # so we can't use something like sub!.
+
```

Comment:
I suggest to use a regex:

```
queue_url = @sqs.get_queue_url({ queue_name: @config[:queue_name] }).queue_url.sub(/^http:\/\//, 'https://')
```

Reply:
We actually need to reassign that variable because the string returned is frozen (i.e., it can't be altered), but I'll refactor that to use a regex.